src/main/java/org/mifos/processor/bulk/API/definition/Simulate.java

```diff
@@ -0,0 +1,13 @@
+package org.mifos.processor.bulk.api.definition;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+// from("rest:post:/simulate").log("Reached Simulation");
+public interface Simulate {
+
+    @PostMapping(value = "/simulate", produces="application/json")
```

Comment:
what is the purpose of this API? can we re-use actuator APIs?

Reply:
@ankita10r can you please address this?? This is the simulate api I migrated from camel to spring because `BD-009 Batch Phased Callback API Test Failure` was failing.

Reply:
This api was meant as a default for testing purposes, this api will return 200 in all scenario therefore prevent flow from getting stuck in callback